docs/operate/mobility/move-arm/frame-how-to.md

```diff
@@ -0,0 +1,96 @@
+
+For example, if you are using a robot arm mounted to a table, it can be convenient to decide that the world frame origin is at the base of the arm, or you can define the world frame origin as one corner of the table that the arm is mounted to.
+
+You do not explicitly configure the world frame.
+You define it implicitly by configuring the frame of a component relative to world.
+
+### Add a frame to your arm
+
+1. Mount the arm in a fixed location.
+1. On your arm's configuration card, click **+ Add frame**.
+1. The card will populate with the following default values:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "orientation": {
+       "type": "ov_degrees",
+       "value": {
+         "x": 0,
+         "y": 0,
+         "z": 1,
+         "th": 0
+       }
+     },
+     "parent": "world",
+     "translation": {
+       "x": 0,
+       "y": 0,
+       "z": 0
+     }
+   }
+   ```
+
+   This [orientation vector](/operate/mobility/orientation-vector/) of `x=0, y=0, z=1, th=0` means the frame is aligned with the world frame, and the `0, 0, 0` translation means the arm's {{< glossary_tooltip term_id="origin-frame" text="origin frame" >}} is coincident with the origin of the world frame.
+   You can think of it like this: The z-axis of the arm's origin frame is parallel to the z-axis of the world frame, and the origin of the arm's origin frame is in the same place as the origin of the world frame.
+
+   If you want to define the world frame as the corner of the table, leave the default values for now, and continue to the next step so that you know which way `x` and `y` point and can edit the configuration accordingly.
+
+1. Click **Save**.
+
+   By configuring your arm's frame like this, you have now defined the world frame as the same as the arm's base frame.
+
+### Figure out which way the arm's axes point
+
+The arm's base frame is now the world frame, but you do not know which way the x, y, and z axes of the arm frame point, unless you happen to be familiar with the arm's [kinematics file](/operate/reference/kinematic-chain-config/).
+
+1. On your arm's configuration card, click the **TEST** section (or use the **CONTROL** tab).
+
+1. Move the arm in each direction and note which way the arm moves.
+   For example, under the **MoveToPosition** section, move the arm in the positive Z direction.
+   If it moves upwards, then you know that the z-axis of the arm frame points upwards.
+1. If the directions do not match your intended world frame, you can edit the arm's `orientation` field to rotate the arm frame to match your intended world frame.
+   For example, if the arm's x-axis points to the right, but you want it to point to the left, you can set the `orientation` field to `"x"=0, "y"=0, "z"=1, "th"=180`.
+
+1. For your reference, mark the X, Y, and Z axes of the world frame on your workspace with tape or a marker.
```

Comment:
It would be great to follow up with a picture of how to mark this as an example. Not a blocker.